Package@swift-5.6.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version:5.6
+import PackageDescription
+
+let package = Package(
+    name: "SwiftFormat",
+    products: [
+        .executable(name: "swiftformat", targets: ["CommandLineTool"]),
+        .library(name: "SwiftFormat", targets: ["SwiftFormat"]),
+        .plugin(name: "SwiftFormatPlugin", targets: ["SwiftFormatPlugin"]),
+    ],
+    targets: [
+        .executableTarget(name: "CommandLineTool", dependencies: ["SwiftFormat"], path: "CommandLineTool"),
+        .target(name: "SwiftFormat", path: "Sources"),
+        .testTarget(name: "SwiftFormatTests", dependencies: ["SwiftFormat"], path: "Tests"),
+        .plugin(name: "SwiftFormatPlugin",
+                capability: .command(
+                    intent: .sourceCodeFormatting(),
```

Comment:
I personally think `swift package format-source-code` is a bit verbose for a command that you'd likely want to run frequently. 

I think something like just `swift package format` is sufficiently clear, and is much more brief / easy to type. That's what we use in Airbnb's SwiftFormat package plugin: https://github.com/airbnb/swift/blob/master/Package.swift#L18

```suggestion
                    intent: .custom(verb: "format", description: "Formats Swift source files using SwiftFormat"),
```

Reply:
Just my personally opinion — I’m not a maintainer, just a contributor. Up to you and @nicklockwood of course.

Reply:
I agree with @calda - `swift package format` seems nicer.

Reply:
> I think something like just 1swift package format` is sufficiently clear, and is much more brief/easy to type. That's what we use in Airbnb's SwiftFormat package plugin

Using plugins from SwiftLint and SwiftFormat from one package will work even if the two plugins share the same intent name because SwiftLint plugin is a build plugin and SwiftFormat plugin is a command plugin.

Plugin execution is impossible when using two command plugins with the same intent name. SPM will throw an error and SPM does not offer a resolution strategy.

```
error: 2 plugins found for ‘format’
Usage: swift package <options> <subcommand>
  See 'package -help' for more information.
```

@nicklockwood  thinking about introducing a custom verb => should it be more specific/unique? For example: `swift plugin swiftformat` ?

Reply:
That makes sense to me.

Reply:
Custom verb `swiftformat` implemented with [2f815d0](https://github.com/nicklockwood/SwiftFormat/pull/1297/commits/2f815d096b9e0744a06ba5a0c7bc2c39a14ac8ab)